app/src/main/java/com/hedvig/app/feature/addressautocompletion/ui/AddressAutoCompleteScreen.kt

```diff
@@ -129,10 +134,31 @@ private fun AddressInput(
             horizontalAlignment = Alignment.CenterHorizontally,
             modifier = Modifier.padding(horizontal = 16.dp, vertical = 8.dp),
         ) {
+            var textFieldValue by remember {
+                val text = viewState.input.rawText
+                mutableStateOf(
+                    TextFieldValue(
+                        text = text,
+                        selection = TextRange(text.length)
+                    )
+                )
+            }
+            LaunchedEffect(viewState.input.selectedDanishAddress) {
+                if (viewState.input.selectedDanishAddress == null) return@LaunchedEffect
+                textFieldValue = textFieldValue.copy(
+                    text = viewState.input.rawText,
+                    selection = TextRange(viewState.input.rawText.length),
+                )
+            }
```

Comment:
This is the functionality that moves the cursor to the end of the word every time an address is selected.